src/contrillers/expenses.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+const getAll = (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
```

Comment:
You dont need to use this headers here because you already declare same functionality in the router